tests/MiSeq_PairedEnd.main.nf.test.snap

```diff
@@ -52,7 +52,6 @@
                 "multiqc/global_report/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                 "multiqc/global_report/multiqc_data/fastqc_sequence_counts_plot.txt",
                 "multiqc/global_report/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
-                "multiqc/global_report/multiqc_data/fastqc_top_overrepresented_sequences_table.txt",
```

Comment:
Do I understand that the tests didn't change, but some outputs were no longer produced?

Reply:
I assume the same

Reply:
That is correct. I checked what these files looked like and they are actually empty for this test. For the other tests, like Novaseq6000 for instance, these files are still there (see https://github.com/Aratz/seqinspector/blob/f28a1e1dcf9a57f187fc5610f5e9e242143d1456/tests/NovaSeq6000.main.nf.test.snap#L52)